pages/linux/mono.md

```diff
@@ -0,0 +1,20 @@
+# mono
+
+> Mono, the open source development platform based on the .NET Framework.
+> More information: <https://www.mono-project.com/docs/>.
+
+- Compile:
```

Comment:
I'm not sure this example belongs here, since we've already got a page for `csc` elsewhere.